From 12f600476e2525f5d86aedfcb9b5535fb2c6e58a Mon Sep 17 00:00:00 2001
From: Anton Bambura <jenneron@protonmail.com>
Date: Mon, 14 Mar 2022 21:44:47 +0200
Subject: [PATCH 1/4] ARM: dts: exynos: peach-pi: add mmc aliases

Add aliases for eMMC, SD card and WiFi, so nubmers will not be
changed anymore.

Signed-off-by: Anton Bambura <jenneron@protonmail.com>
Tested-by: Valentine Iourine <iourine@iourine.msk.su>
---
 arch/arm/boot/dts/exynos5800-peach-pi.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/exynos5800-peach-pi.dts b/arch/arm/boot/dts/exynos5800-peach-pi.dts
index 77013ee58..aa1d14af4 100644
--- a/arch/arm/boot/dts/exynos5800-peach-pi.dts
+++ b/arch/arm/boot/dts/exynos5800-peach-pi.dts
@@ -29,6 +29,10 @@ / {
 	aliases {
 		/* Assign 20 so we don't get confused w/ builtin ones */
 		i2c20 = &i2c_tunnel;
+
+		mmc0 = &mmc_0; /* eMMC */
+		mmc1 = &mmc_2; /* SD */
+		mmc2 = &mmc_1; /* WiFi */
 	};
 
 	backlight: backlight {
-- 
2.25.1

